### ISSUE TITLE
Rotated axis titles

### ISSUE THREAD

**Issue by objorke** [oxyplot/oxyplot#34]
Add a "TitleAngle" for the axes. ("Angle" is used for the tick labels)

**Comment by objorke** [comment]
Is #22 a better solution? I think the rotated titles is only needed for vertical axes.

**Comment by LukaBedi** [comment]
Hi Objorke, I really love your work and oxyplot is amazing :) i have recently come across the need to rotate the Y-axis label 90° and so i was wondering if issue #34 is solved ?

**Comment by objorke** [comment]
No, this issue is open. The question is if it should be implemented as described in #22.
